test/controllers/GuidanceControllerSpec.scala

```diff
@@ -90,23 +104,36 @@ class GuidanceControllerSpec extends BaseSpec with GuiceOneAppPerSuite {
       mockGuidanceService,
       stubMessagesControllerComponents()
     )
+
+    val pec = PageEvaluationContext(
+                page,
+                processId,
+                Map(),
+                Some("/hello"),
+                Text(),
+                processId,
+                "hello",
+                LabelCache(),
+                None,
+                None
+              )
   }
 
   "Calling a valid URL path to a Question page in a process" should {
 
     "return an OK response" in new QuestionTest {
       MockGuidanceService
-        .getPageContext(processId, path, processId, None)
-        .returns(Future.successful(Right(PageContext(expectedPage, Some("/"), Text(Nil, Nil), processId, processCode))))
+        .getPageContext(processId, path, processId)
+        .returns(Future.successful(Right(PageContext(expectedPage, processId, Some("/"), Text(Nil, Nil), processId, processCode))))
```

Comment:
Could we create a value for sessionId and use it when creating the PageContext instances in the tests?